examples/tutorials/self-paced-training/welcome.ipynb

```diff
@@ -22,18 +22,20 @@
    "metadata": {},
    "source": [
     "# Course Goals\n",
-    "In this course, our primary goal is to **introduce the fundamentals of NVIDIA FLARE to researchers, scientists, and developers**.\n",
+    "In this course, our primary goal is to **introduce the fundamentals of federated learning with NVIDIA FLARE**.\n",
```

Comment:
this file at line 16 we have a typo:
"Welcome to the **sef-paced** training Course **Practical Federated Learning with NVIDIA FLARE**!"

=> self-paced